src/screens/DetailsScreen/DetailsScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import {RouteProp} from '@react-navigation/native';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {
+  SafeAreaView,
+  StatusBar,
+  Text,
+  useColorScheme,
+  View,
+} from 'react-native';
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+import {StackParamList} from 'types/navigation';
+import {CityItem} from '../../components';
+import styles from './DetailsScreenStyles';
+
+type DetailsProps = NativeStackScreenProps<StackParamList, 'Details'>;
+
+const DetailsScreen: React.FC<DetailsProps> = ({route}) => {
+  const {humidity, wspeed, pressure, cloud} = route.params;
+  const isDarkMode = useColorScheme() === 'dark';
+
+  const backgroundStyle = {
```

Comment:
no need to define additional variable - you can just use this directly:
`<SafeAreaView style={{backgroundColor: isDarkMode ? Colors.darker : Colors.lighter}}>`

Reply:
we should avoid creating an object inside react component - because each render it will be a new one
just add variable:
`const backgroundColor = isDarkMode ? Colors.darker : Colors.lighter`
and re-use in styles for SafeAreaView and in StatusBar